pkg/virt-handler/vm.go

```diff
@@ -1555,6 +1556,15 @@ func (d *VirtualMachineController) updateDomainFunc(old, new interface{}) {
 }
 
 func (d *VirtualMachineController) heartBeat(interval time.Duration, stopCh chan struct{}) {
+	// This is a temporary workaround until k8s bug #66525 is resolved
```

Comment:
do we need to find the correct path on each heartBeat or can it be done once outside of this function?

Reply:
it's called only once, outside the loop [here](https://github.com/kubevirt/kubevirt/pull/2639/files/a66097c601e994e4a12b9f0061a56cc069e42479#diff-27207346391eaa19190c76760eb8851fR1568) :)

Reply:
ah, sorry, I was sure that heartBeat is being called from a loop. Sorry :)